### ISSUE TITLE
Trouble replicating results for ResNet18 on ImageNet 1k 

### ISSUE THREAD

**Issue by dyelax** [tornadomeet/ResNet#50]
I'm trying to replicate your results for ResNet18 on ImageNet 1k. I'm using the default hyperparameters in your `train_resnet.py` and all of the same augmenters (for aug-level=2), but can't achieve the accuracy that you report. The most obvious issue is that if I use your default learning rate of 0.1, the net fails to learn anything, even though I'm using the same batch size (256), momentum (0.9), weight decay (0.0001), etc. I have to set a learning rate to 0.001 for it to start learning well, but even then, the accuracy tops out at about 80% for Top 5 and 55% for Top 1.

1. I see in the code that these are the default parameters for ResNet50. Did you change any of them for ResNet18?

2. Is the data that you are using standard ImageNet, or did you modify it before training (by normalizing, etc.)?

**Comment by JaworMichal** [comment]
Hi, I've recently had a problem with the replication of results for a pre-trained DeiT on the ImageNet validation part and the thing is I've had a slightly different version of ImageNet. Maybe your problems with training are due to a different dataset. So make sure can at least replicate the results for a pre-trained net.
